test/e2e/multi-select.test.js

```diff
@@ -9,8 +9,115 @@ describe('vl-multi-select', async () => {
         return vlMultiSelectPage.load();
     });
 
+    it('Als gebruiker kan ik de values van de multiselect opvragen', async () => {
+        const multiselect = await vlMultiSelectPage.getVoorgeselecteerdeMultiselect();
+        await assert.eventually.include(multiselect.values(), 'Brussels', 'Ghent');
+    });
+
+    it('Als gebruiker kan ik options die voorgedefinieerd zijn, opvragen', async () => {
+        const multiselect = await vlMultiSelectPage.getVoorgeselecteerdeMultiselect();
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Bruges');
+    });
+
+    it('Als gebruiker kan ik een optie selecteren en deze zal dan in de combobox getoond worden', async () => {
+        const multiselect = await vlMultiSelectPage.getStandardMultiselect();
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'Germany');
+        await multiselect.selectByValue('Germany');
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Germany');
+    });
+
+    it('Als gebruiker kan ik een waarde uit de multiselect selecteren via value', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetOnbeperkteResultaten();
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'aakstraat');
+        await multiselect.selectByValue('aakstraat');
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'aakstraat');
+    });
+
+    it('Als gebruiker kan ik een waarde uit de multiselect selecteren via text', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetOnbeperkteResultaten();
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'aalmoezenierstraat');
+        await multiselect.selectByText('Aalmoezenierstraat');
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'aalmoezenierstraat');
+    });
+
+    it('Als gebruiker kan ik een gekozen optie verwijderen', async () => {
+        const multiselect = await vlMultiSelectPage.getGegroepeerdeMultiselect();
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Bruges');
+        await multiselect.delete('Bruges');
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'Bruges');
+    });
+
```

Comment:
Test voor `Multiselect met gegroepeerde opties` ontbreekt. Daar kan je testen dat de opties gegroepeerd zijn met tussentitels.

Reply:
Toegevoegd.

Reply:
De test beschrijving aanpassen en ik zou in de test de groepering controleren en ook verifiëren dat de tussentitels gerenderd worden.